docs/docs/ref-05-events.ko-KR.md

```diff
@@ -139,7 +139,6 @@ DOMEventTarget relatedTarget
 
 이 포커스 이벤트는 폼 엘리먼트뿐만 아니라 모든 React DOM 엘리먼트에서 작동합니다.
 
-<a name="form-events"></a>
```

Comment:
We don't use Kramdown, we use Redcarpet. The header generation is done in a custom plugin (https://github.com/facebook/react/blob/master/docs/_plugins/header_links.rb) which is pretty naive (only supports a small number of characters).

When making changes like this, please run the server and ensure they work. This change won't work (as @marocchino said) without also changing how we generate the header links.